examples/e2e/edge-native/resource_cluster.tf

```diff
@@ -19,19 +17,9 @@ resource "spectrocloud_cluster_edge_native" "cluster" {
     name                    = "master-pool"
 
     edge_host {
-      host_uid  = spectrocloud_appliance.appliance0.uid
-      static_ip = "126.10.10.23"
+      host_uid  = var.edge_id
+      #static_ip = "123.45.67.89"
```

Comment:
Let's put some description about use-cases where static IP may be optional.

Reply:
Overall I would wait for overlay feature to land before we revisit this example.